Source/C++/Core/Ap4Atom.cpp

```diff
@@ -36,7 +36,7 @@
 #include "Ap4AtomFactory.h"
 #include "Ap4Debug.h"
 #include "Ap4UuidAtom.h"
-
+#include <cstdint>
```

Comment:
What type specifically what missing?

Reply:
It was an issue in a previous commit, but it is solved now
I will remove it now